src/state/lists/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { minVersionBump, TokenList, VersionUpgrade } from '@uniswap/token-lists'
+
+export function shouldAcceptMinorVersionUpdate(
+  listUrl: string,
+  current: TokenList,
+  update: TokenList,
+  targetBump: VersionUpgrade.PATCH | VersionUpgrade.MINOR
+): boolean {
+  const min = minVersionBump(current.tokens, update.tokens)
+  // automatically update minor/patch as long as bump matches the min update
```

Comment:
nit: Always start comment with a capital letter and end with a period (sentence)